### ISSUE TITLE
Suggestion: get rid of ddhcpdctl and use uc instead saving space in the image

### ISSUE THREAD

**Issue by christf** [sargon/ddhcpd#11]
We could replace ddhcpctl with uc from https://github.com/christf/uc.git saving some space in the image.
uc is more generic and also used by prefixd and can be used for diagnostics together with l3roamd.

**Comment by sargon** [comment]
Yes, we could redesign the socket that way. But I would still provide a then adopted ddhcpdctl here.
But then we don't need to package in gluon.
